reporter/otlp_reporter.go

```diff
@@ -515,16 +521,10 @@ func (r *OTLPReporter) getProfile() (profile *profiles.Profile, startTS, endTS u
 			Unit: int64(getStringMapIndex(stringMap, "nanoseconds")),
 		},
 		Period: 1e9 / int64(r.samplesPerSecond),
-		// LocationIndices - Optional element we do not use.
-		// AttributeTable - Optional element we do not use.
 		// AttributeUnits - Optional element we do not use.
 		// LinkTable - Optional element we do not use.
 		// DropFrames - Optional element we do not use.
 		// KeepFrames - Optional element we do not use.
-		// TimeNanos - Optional element we do not use.
-		// DurationNanos - Optional element we do not use.
-		// PeriodType - Optional element we do not use.
-		// Period - Optional element we do not use.
 		// Comment - Optional element we do not use.
 		// DefaultSampleType - Optional element we do not use.
```

Comment:
I'd also remove all of these, they're just noise.

Reply:
No strong feelings here. These comments helped to implement the protocol and to check which fields are implemented and which not. When working on this change, I just noticed, that some of these comments are no longer correct.